### ISSUE TITLE
Data.table warning

### ISSUE THREAD

**Issue by nilsreiter** [quadrama/DramaAnalysis#150]
```R
text <- loadDrama("11f9k.0")
utteranceStatistics(text)
```

leads to this warning:

```
Warning message:
In `[.data.table`(text, , `:=`(dl, .N), .(corpus, drama)) :
  Invalid .internal.selfref detected and fixed by taking a (shallow) copy of the data.table so that := can add this new column by reference. At an earlier point, this data.table has been copied by R (or was created manually using structure() or similar). Avoid names<- and attr<- which in R currently (and oddly) may copy the whole data.table. Use set* syntax instead to avoid copying: ?set, ?setnames and ?setattr. If this message doesn't help, please report your use case to the data.table issue tracker so the root cause can be fixed or this message improved.

```
